lib/features/search/presentation/ui/widgets/search_bottom_drawer.dart

```diff
@@ -0,0 +1,234 @@
+              }
+              return AnimatedPositioned(
+                  curve: Curves.easeInOut,
+                  duration: Duration(milliseconds: 200),
+                  left: 0,
+                  bottom: -232,
+                  child: DrawerContents());
+            },
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class DrawerContents extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    double width = MediaQuery.of(context).size.width;
+
+    return ClipRRect(
+      borderRadius: BorderRadius.all(Radius.circular(10.0)),
+      child: Container(
+        color: Colors.redAccent,
+        width: width,
+        height: height / 3 + 60,
+        child: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 2.0),
+          child: Column(
+            children: <Widget>[
+              Icon(
+                Icons.keyboard_arrow_up,
+                size: 20,
+              ),
+              Text(
+                "Place Details",
+                style: TextStyle(
+                    fontSize: 17,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.white),
+              ),
+              PlaceDetails(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class PlaceDetails extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DrawerContentsBloc, DrawerContentsState>(
+      builder: (BuildContext context, DrawerContentsState state) {
+        if (state is DrawerContentsLoaded) {
+          return PlaceCard(
+              placePhoto: state.photo, placeEntity: state.placeEntity);
+        } else {
+          return Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Text(
+                "No places searched yet...",
+                style: TextStyle(
+                  fontSize: 20,
+                  fontWeight: FontWeight.bold,
+                ),
+              ));
+        }
+      },
+    );
+  }
+}
+
+class PlaceCard extends StatelessWidget {
+  final File placePhoto;
+  final PlaceEntity placeEntity;
+
+  const PlaceCard({Key key, this.placePhoto, this.placeEntity})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final navScreenBloc = BlocProvider.of<NavigationScreenBloc>(context);
+    final photoBytes = placePhoto.readAsBytesSync();
+
+    return Row(
+      children: <Widget>[
+        Container(
+          padding: EdgeInsets.only(top: 10.0, right: 10.0),
+          height: 140,
+          width: 350,
+          child: Card(
+            elevation: 5,
+            child: Row(
+              children: <Widget>[
+                Container(
+                    child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Image.memory(
+                    photoBytes,
+                    width: 140,
```

Comment:
Remove this height and width? Wrap this container in expanded.

Reply:
Wrapped it in expanded and removed the width. However, height needs to be maintained due to the drawer working on screen offset.